bddisasm/bdx86_decoder.c

```diff
@@ -3378,22 +3378,22 @@ NdFindInstruction(
         break;
     }
 
-    while ((!stop) && (ND_NULL != pTable))
```

Comment:
Adding so many `goto`s inside the `while` loop makes the code quite hard to follow. I wouldn't make such changes to the decoder loop, unless they offer a significant benefit.

Reply:
This commit alone is about 2% faster. I'd call that significant.

Reply:
It may do so on Linux, but it brings no benefit on Windows (in fact, I see a small performance downgrade with this modification). Combined with the more hard to follow code, I see no reason to merge the changes to the `NdFindInstruction` function at this moment.
The other modifications seem OK though.

Reply:
I've left this bit out for now.